IPython/core/display.py

```diff
@@ -599,6 +602,9 @@ def __init__(self, data=None, url=None, filename=None, expanded=False, metadata=
         if kwargs:
             self.metadata.update(kwargs)
         super(JSON, self).__init__(data=data, url=url, filename=filename)
+    
+    def __repr__(self):
+        return self.data
```

Comment:
Thanks for this !

Shouldn't the JSON text representation return something more like `Json.dumps(self.data, indent=2)` ?

Reply:
IDK how to make that optional for jsonlines or similar; so this is fine as-is imho

Reply:
Unless it could be `__str__` instead of `__repr__`?